src/main/java/net/glowstone/entity/GlowLivingEntity.java

```diff
@@ -854,11 +854,9 @@ public void damage(double amount, Entity source, DamageCause cause) {
         }
 
         // play sounds, handle death
-        if (health > 0) {
-            Sound hurtSound = getHurtSound();
-            if (hurtSound != null && !isSilent()) {
-                world.playSound(location, hurtSound, getSoundVolume(), getSoundPitch());
-            }
+        Sound hurtSound = health > 0 ? getHurtSound() : getDeathSound();
```

Comment:
What is the reason for this change? Death sound is already played here: 

https://github.com/Pr0methean/Glowstone/blob/eae90f1453fc76a33a59317a3877b3776e6a247e/src/main/java/net/glowstone/entity/GlowLivingEntity.java#L724-L729

Reply:
Reverted.